### ISSUE TITLE
Minikeepass: Password and keyfile simultaneously does not work

### ISSUE THREAD

**Issue by derlefti** [MiniKeePass/MiniKeePass#622]
Hi,
I am using keepass for Windows and Minikeepass for iOS. I tried to synchronize via dropbox (encrypted via boxcryptor). I am aware of the fact that Minikeepass does not synchronize automatically and that you have to manually copy the files to/from dropbox.
In principle, everything works fine as long as I define EITHER a password OR a keyfile for opening the database. For some reason minikeepass is not able to open a database which is secured by both (Message: "Could not open database"). In Keepass for Windows it works fine. Can anyone help me with this?

**Comment by gpborges** [comment]
I'm also facing the exact same issue using latest iOS 12.1 on an iPhone 8 Plus and latest version of MiniKeepass I just installed from App Store.

**Comment by pinmu** [comment]
no problem on iOS 12.0.1 so far.

**Comment by mickalvey** [comment]
Yes, there is a problem for sure using one or the other or both. The developers could provide more detail on how to use or NOT use the combinations for passwords. It doesn't take kindly to changing from one to the other....like password on file to key file.? This app needs a lot of work to make it compatible with the other developers of the similar code KeepassX and KeepassXC, etc. Not sure if Minikeepass is even related developer-wise. I say if you don't like the usage features or lack of, then find another solution
